### ISSUE TITLE
Is there a way to generate swagger from a flasgger app? 

### ISSUE THREAD

**Issue by ashee** [rochacbruno/flasgger#138]
I am having trouble finding documentation on how to reverse generate swagger.yaml or swagger.json from a flasgger app. Is this supported? 

Ideally, I would like to hit an endpoint similar to /apidocs/ such as /apidocs/swagger.yaml that would generate a swagger yaml.

**Comment by ashee** [comment]
I discovered that the javascript app is invoking this endpoint /apispec_1.json to get a swagger spec in json. Is that a stable URI?

**Comment by rochacbruno** [comment]
@ashee yes, specs will be generated in json in `/apispec_1.json` and if you have more than one spec in the list it will be `_2.json` and so on..

**Comment by vringar** [comment]
Do you think it's common enough of a usecase to mention this in README?
Would you accept a PR to mention this there?
